R/step-mutate.R

```diff
@@ -56,6 +56,19 @@ mutate_with_braces <- function(mutate_vars) {
 #' @param ... <[data-masking][dplyr::dplyr_data_masking]> Name-value pairs.
 #'   The name gives the name of the column in the output, and the value should
 #'   evaluate to a vector.
+#' @param .keep `r lifecycle::badge("experimental")`
```

Comment:
Might need some note here, maybe just at the end, explaining that this is based on the symbols present in the generated `data.table` code. Stuff (rarely used) like `get()` and `eval()` will result in different output than dplyr. For example, dplyr returns a three-column tibble with `mutate(mtcars, y = eval(parse(text = "wt + hp")), .keep = "used")`